src/input_common/sdl/sdl_impl.h

```diff
@@ -66,8 +82,26 @@ class SDLState : public State {
     /// Needs to be called before SDL_QuitSubSystem.
     void CloseJoysticks();
 
-    /// Returns a custom name for specific controllers because the default name is not correct
-    std::string GetControllerName(SDL_GameController* controller) const;
+    /// Returns the default button bindings list for generic controllers
+    const ButtonBindings GetDefaultButtonBinding() const;
```

Comment:
```suggestion
    ButtonBindings GetDefaultButtonBinding() const;
```
Given this just returns by value. Ditto for the other functions below